src/app/space/create/codebases/codebases-item-workspaces/codebases-item-workspaces.component.less

```diff
@@ -23,12 +23,20 @@
 
 .workspace-select {
   height: 32px;
-  margin-right: 5px;
-  width: auto;
   display: initial;
 }
 
 .link-icon {
   height: 1em;
   margin-right: -36px;
 }
+
+.no-gutters {
+  margin-right: 0;
+  margin-left: 0;
+}
+
+.no-gutters > [class*='col-'] {
```

Comment:
I think we already have some utility classes for no-gutters in `assets/stylesheets/shared/_layout.less`. This exact same implementation actually.